pkg/controller/mpi_job_controller_test.go

```diff
@@ -233,16 +260,16 @@ func (f *fixture) newController(gangSchedulerName string, clock clock.WithTicker
 	return c, i, k8sI
 }
 
-func (f *fixture) run(mpiJobName string) {
-	f.runWithClock(mpiJobName, clock.RealClock{})
+func (f *fixture) run(mpiJobName, gangSchedulerName string) {
```

Comment:
should the scheduler name be part of the fixture?

Reply:
Sure.

Reply:
Done.